frontend/app/views/support/Pricing.scala

```diff
@@ -23,7 +23,7 @@ case class Pricing(yearly: Price, monthly: Price) {
     case Month() => monthly
   }
 
-  def getPhrase(period : BillingPeriod){
+  def getPhrase(period : BillingPeriod): String = {
```

Comment:
@svillafe - just so you know, until this change, this function was returning `Unit` rather than `String` - unless a Scala function has `=` in it's declaration, it will always return `Unit`!